src/rust/generators/sysmon-subgraph-generator/src/lib.rs

```diff
@@ -0,0 +1,6 @@
+#![type_length_limit = "1334469"]
```

Comment:
Are these needed anymore? As I recall they were some workaround to a weird issue in Rust a while back.